src/main/java/bio/terra/service/dataset/flight/datadelete/DatasetDataDeleteFlight.java

```diff
@@ -37,8 +39,8 @@ public DatasetDataDeleteFlight(FlightMap inputParameters, Object applicationCont
         // get data from inputs that steps need
         String datasetId = inputParameters.get(JobMapKeys.DATASET_ID.getKeyName(), String.class);
 
-        RetryRuleRandomBackoff lockDatasetRetry =
-            new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
+        RetryRule lockDatasetRetry = getDefaultRandomBackoffRetryRule(appConfig.getMaxStairwayThreads());
+        RetryRule unlockDatasetRetry = getDefaultRandomBackoffRetryRule(appConfig.getMaxStairwayThreads());
```

Comment:
There is no problem reusing retry rules. They were designed to be reusable within a Flight.
- only one Step is run at a time
- the RetryRule has an initialize method that is called at the start of a step to set the values

So your change is not needed.

Reply:
Ah ok.  Misunderstanding on my part.  I'll undo that but